src/locale/languages.ts

```diff
@@ -54,6 +55,7 @@ interface AppLanguageConfig {
 export const APP_LANGUAGES: AppLanguageConfig[] = [
   {code2: AppLanguage.en, name: 'English'},
   {code2: AppLanguage.an, name: 'Aragonés – Aragonese'},
+  {code2: AppLanguage.ar, name: 'اَلْعَرَبِيَّةُ – Arabic'},
```

Comment:
```suggestion
  {code2: AppLanguage.ar, name: 'العربية – Arabic'},
```
No need to all these diacritics; keep it clean.

Reply:
Thanks! :)